### PR TITLE
USBAT sign up, login , logout

### PR DESCRIPTION
**Pull request by richiebeans**
Created skeleton for landing page (HTML/CSS)
Log-in functionality working
Still to come: Log out functionality, recently added flats, search functionality.

### CODE REVIEW

**Comment by f3r** [comment]
- working navbar with user auth functionnalities

**Comment by f3r** [comment]
- landing page basic frame

**Comment by f3r** [comment]
Signup should also have the same style as the rest of the app, when i click on `Don't have an account? Register now!`, I'm taken to an unstyled page. Can you guys fix that?

**Comment by f3r** [comment]
Why you ask for first name but not last name?

**Comment by f3r** [comment]
When trying to signup, I get a `Failed to load resource: the server responded with a status of 403 (Forbidden)` and the following rails server log:

```
Started POST "/auth" for ::1 at 2015-08-25 11:18:38 +0800
Processing by DeviseTokenAuth::RegistrationsController#create as HTML
  Parameters: {"email"=>"fer@fer.com", "password"=>"[FILTERED]", "password_confirmation"=>"[FILTERED]", "first_name"=>"Fer", "confirm_success_url"=>"http://localhost:3000/#/signup", "config_name"=>"default", "registration"=>{"email"=>"fer@fer.com", "password"=>"[FILTERED]", "password_confirmation"=>"[FILTERED]", "first_name"=>"Fer", "confirm_success_url"=>"http://localhost:3000/#/signup", "config_name"=>"default"}}
Unpermitted parameters: confirm_success_url, config_name, registration
Unpermitted parameters: confirm_success_url, config_name, registration
Unpermitted parameters: confirm_success_url, config_name, registration
   (1.3ms)  BEGIN
   (1.7ms)  SELECT COUNT(*) FROM "users" WHERE "users"."provider" = $1 AND "users"."email" = $2  [["provider", "email"], ["email", "fer@fer.com"]]
   (1.2ms)  ROLLBACK
Completed 403 Forbidden in 280ms (Views: 0.8ms | ActiveRecord: 19.9ms)
```

**Comment by richiebeans** [comment]
The sign up page was included in a seperate pull request, not this one which is only for the landing page.  The current sign up page was just a dummy to test POST USER. Raph is currently working on completing the actual sign up page and will address all your comments in the pull request for the sign up page. Let me know if you need me to elaborate further.

**Comment by f3r** [comment]
@richiebeans @raphaeldieu let me know when this is ready for review

File: app/assets/javascripts/angular/controllers/auth/signup.js.erb [hunk 1]
```diff
@@ -1,9 +1,36 @@
...
+                Id: 3,
+                Name: 'Unemployed'
+            }, {
+                Id: 4,
+                Name: 'Retired'
+            }];
+
+            
+  $scope.GetOccupation = function (occupation) {
+      // var occupationId = $scope.ddlOccupations;
```
Comment (f3r) [inline app/assets/javascripts/angular/controllers/auth/signup.js.erb]: did you guys managed to get occupation working? why is there commented code?

**Comment by richiebeans** [comment]
Yes authenticate is working and sorry for the commented out code, this has been removed.

File: app/assets/templates/static_pages/index.html [hunk 2]
```diff
@@ -1,31 +1,120 @@
...
+          <label class="sr-only" for="">Enter search terms</label>
+          <input type="search" class="form-control" id="front-search" placeholder="Find a room">
+          </input>
+        </div>
+        <button type="submit" class="btn btn-default"><span class="glyphicon glyphicon-search"></span>
+      </form>
+    </li>        
+    <!-- list room -->
+    <li class="pull-right">
+      <a href="" id="listmyroom">LIST MY ROOM</a>
```
Comment (f3r) [inline app/assets/templates/static_pages/index.html:23]: don't add stuff till you have it!

File: app/assets/templates/static_pages/index.html [hunk 3]
```diff
@@ -1,31 +1,120 @@
...
+  <div class="text-vertical-center wrapper">
+    <div class="title-txt">
+      <h1>Your next roommate is just a click away.</h1>
+    </div>
+  </div>
+  <!-- left box -->
+  <div class="text-vertical-center wrapper col-lg-5 left-col">
+    <div class="front-left">
+      <form>
+        <table>
```
Comment (f3r) [inline app/assets/templates/static_pages/index.html:67]: tables, really? ... sigh

File: config/database.yml [hunk 4]
```diff
@@ -17,5 +17,5 @@ test:
   database: roomi_test
 
 production:
-  <<: *default
-  database: db/production.sqlite3
\ No newline at end of file
+  # <<: *default
```
Comment (f3r) [inline config/database.yml:20]: @raphaeldieu no no no.... pls no more comments

**Pull request merged**